tools/SchemaDifference/SchemaDifferenceRunner.js

```diff
@@ -165,6 +169,11 @@ function versionCompare(version1, version2) {
   const [read1, write1, minor1] = version1.split(".");
   const [read2, write2, minor2] = version2.split(".");
 
+  // If schema version not specified
+  if ([read1, write1, minor1].includes(undefined)) {
+    return;

```

Comment:
it seems a bit odd to return nothing here

Reply:
fixed